benchexec/tools/tabol.py

```diff
@@ -0,0 +1,47 @@
+"""
+BenchExec is a framework for reliable benchmarking.
+This file is part of BenchExec.
+Copyright (C) 2007-2015  Dirk Beyer
+All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import benchexec.result as result
+import benchexec.util as util
+import benchexec.tools.template
+
+class Tool(benchexec.tools.template.BaseTool):
+
+    def executable(self):
+        return util.find_executable('tabol.jar')
```

Comment:
Did you test that this works? This should require `tabol.jar` to be executable, which JARs usually aren't.

What is the output of the [test](https://github.com/sosy-lab/benchexec/blob/master/doc/tool-integration.md#testing-the-tool-integration)?

Reply:
I changed the jar file to a shell file now! then it works.